js/structure.js

```diff
@@ -83,48 +55,27 @@ var SnakesAndLadder = (function () {
     let snakes;
     let ladders;
 
-    /**
-     * Constructor
-     */
     function SnakesAndLadder() {
         snakes = [];
         ladders = [];
     }
 
-    /**
-     * Add snake.
-     * @param {Snake} snake 
-     */
     SnakesAndLadder.prototype.addSnake = function (snake) {
         snakes.push(snake);
     }
 
-    /**
-     * Add ladder.
-     * @param {Ladder} ladder 
-     */
     SnakesAndLadder.prototype.addLadder = function (ladder) {
         ladders.push(ladder);
     }
 
-    /**
-     * Returns the array of all snakes.
-     */
     SnakesAndLadder.prototype.getSnakes = function () {
         return snakes;
     }
 
-    /**
-     * Returns the array of all ladders.
-     */
     SnakesAndLadder.prototype.getLadders = function () {
         return ladders;
     }
 
-    /**
-     * Returns the new position if the ladder jumps.
-     * @param {int} position 
-     */
     SnakesAndLadder.prototype.getLadderJump = function (position) {
         for (let x = 0; x < ladders.length; x++) {
```

Comment:
Loop variable name can be descriptive so that the code becomes further readable.